pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -77,6 +77,7 @@ func NewRepoSyncReconciler(clusterName string, reconcilerPollingPeriod, hydratio
 			scheme:                  scheme,
 			reconcilerPollingPeriod: reconcilerPollingPeriod,
 			hydrationPollingPeriod:  hydrationPollingPeriod,
+			syncKind:                "RepoSync",
```

Comment:
Ironically, none of the core k8s objects have constants for the kinds. I could make a constant, but I'm not sure where to put it. I could put it in the CRD api packages, but kubebuilder doesn't generate it or have a convention for it. I could make a constant for the reconciler-manager, but that seems kind of silly, since the reconciler uses it too.

Reply:
done